nrcatalogtools/maya.py

```diff
@@ -31,8 +31,6 @@ def __init__(self, catalog=None, use_cache=True, verbosity=0, **kwargs) -> None:
         self.waveform_data_url = utils.maya_catalog_info["data_url"]
         self.waveform_data_dir = utils.maya_catalog_info["data_dir"]
 
-        self._add_paths_to_metadata()
```

Comment:
Why are you deleting this call? It is needed to keep metadata/waveform data filepaths available in the catalog object's attributes.